### ISSUE TITLE
Saving files is buggy

### ISSUE THREAD

**Issue by nevadascout** [nevadascout/Dash#73]
Files save properly, but doing so leaves the editor in a buggy state.

It would be good to get it working so that the "save all" and "close all tabs" features can be properly implemented.

This includes changing the "file dirty" prompt to be "close without saving".

**Comment by trdwll** [comment]
Close all tabs has been implemented in 1.3 dev. 

What do you mean a buggy state? I can save just fine.

**Comment by nevadascout** [comment]
Close all tabs is partially implemented in 1.3-dev -- but it doesn't do any file state checking before closing, so it won't prompt you to save a modified file.

I can't remember exactly what was buggy, but I'll take another look.

**Comment by trdwll** [comment]
Yeah, I forgot to add in checking on the close all tabs. I'll do that later aswell, before I push.
